index.html

```diff
@@ -4,9 +4,55 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+    <link rel="stylesheet" href="./styles.css">
+    <title>Leigh-Ann's Website</title>
 </head>
 <body>
+  <h1>About Leigh-Ann</h1>
+  <p>This is website that's all about Leigh-Ann, a newb at coding. </p>
+  <ul>
+      <li><a href="#basics">Basics</a></li>
+      <li><a href="#favorite">Favorites</a></li>
+      <li><a href="#">Glass</a></li>
+  </ul> 
+  <hr>
+  <div id="basics">
+      <h2>Basics</h2>
+      <p>Leigh-Ann is originally from Pittsburgh, Pennsylvania. 
+    She attended the Ohio State University (<em>go bucks!</em>), and recieved a BFA in Glass Construction.
+    After graduation, Leigh-Ann excavated stained glass from a medival archeology site in Ireland.
+    In 2015, Leigh-Ann relocated to the PNW for a job oppurtunity at Pilchuck Glass School.
+    After spending several summers making glass art in the woods,</p>  
+    </div> 
```

Comment:
make sure indentation is correct.

Reply:
Yes you indent things inside the div tags. Also you will have <p> tags on new lines, like <div> tags.